### PR TITLE
ruby project setup

### LINKED ISSUE #3: Set up new ruby project
# Why
Set up a new ruby project with all base files

### PR DESCRIPTION
**Pull request by thiantonello**
Closes #3

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+source "https://rubygems.org"
+
+gem 'rspec'
+gem 'rake'
+gem 'rubocop'
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# gem "rails"
```
Comment (brunohkbx) [inline Gemfile:11]:
```suggestion
```

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+/test/version_tmp/
+/tmp/
+
+## Environment normalization
+/.bundle/
+/vendor/bundle
+/lib/bundler/man/
+
+# Used by RuboCop. Remote config files pulled in from inherit_from directive.
+.rubocop-https?--*
```
Comment (brunohkbx) [inline .gitignore:19]: EOF

File: Rakefile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  task.fail_on_error = false
+end
+
+task :run do
+  ruby 'lib/program.rb'
+end
+
+task :test do
+  ruby 'spec/spec_helper.rb'
+end
```
Comment (brunohkbx) [inline Rakefile:16]: EOF

File: fixtures/game_short.log [hunk 4]
```diff
@@ -0,0 +1,674 @@
+  0:00 ------------------------------------------------------------
```
Comment (m-pereira) [inline fixtures/game_short.log:1]: Just for test, you don't need to comment all origin log file, since it's huge.

Comment (thiantonello) [reply fixtures/game_short.log:1]: I have implemented your suggestion. Thank you for the tip!

**Pull request merged**